KrotovaEY/lab1/Hello/Source.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <conio.h> 
+using namespace std;
+
+int main()                       
+{        
+	setlocale(0, "");
+	float left, right;
+	cout << "������� ����� �������:";
+	cin >> left;
+	cout << "������� ������ �������:";
+	cin >> right;
+	char sign;
+	cout << "������� ����:";
+	cin >> sign;
+	float result;
+	switch (sign) {
+	case '+': result = left + right;
+		break;
+	case'-': result = left - right;
+		break;
+	case'*': result = left * right;
+		break;
+	case'/': result = left / right;
+		break;
+	}
+	if (right==0 && sign=='/')
+	{
+		cout << "�� ���� ������ ������";
+	} 
+	else if (sign!='+'&&sign!='-'&&sign!='*'&&sign!='/')

```

Comment:
As i mentioned in previous code review, you are checking sign variable twice. First one is switch statement and the second - here.
Try to solve it in another way